core/types/backwards_compat_diffpkg.libevm_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2024-2025 the libevm authors.
```

Comment:
nit feel free to ignore, but our linter should complain theoretically, since the file is created in 2025 🤷 

```suggestion
// Copyright 2025 the libevm authors.
```